CMakeLists.txt

```diff
@@ -2,24 +2,26 @@ cmake_minimum_required (VERSION 2.6)
 cmake_policy(SET CMP0048 NEW)
 project (LIBUAST
   VERSION 0.0.0
-  LANGUAGES C)
+  LANGUAGES CXX C)
+
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11" CACHE STRING "" FORCE)
```

Comment:
Tried that, fails with travis version of CMake.

Reply:
Hmmm, the problem must be elsewhere I'd say, since that's what we use in both examples and testsuite (i.e.: https://github.com/bblfsh/libuast/blob/master/examples/CMakeLists.txt#L3)

Reply:
I'll take a second look.

Reply:
Fixed. I noticed that file was different in my branch, so I was using a version before the Optional removal and that's why it was really failing. This also fixed the `LANGUAGES CXX C` above.